### ISSUE TITLE
Non-Image content-type returned for animated gif

### ISSUE THREAD

**Issue by eostermueller** [atmos/camo#114]
I'm getting the message "Non-Image content-type returned" with this animated gif: http://recordit.co/iEMpbBUfQ0

I added some markdown.  In the following line, it renders incorrectly, exactly as it renders on my github wiki page:
  ![testing](http://recordit.co/iEMpbBUfQ0)

to this github wiki page:
  https://github.com/eostermueller/performanceGolf/wiki/Install-and-Run

...which generated this url:
https://camo.githubusercontent.com/31065e879f38322ec91c2fb8f6a8479362bfd161/687474703a2f2f7265636f726469742e636f2f69454d70624255665130

Any thoughts on how to make this work?

Thank you,
--Erik

**Comment by sweepyoface** [comment]
GitHub uses a custom version of Camo, have you tried testing with your own copy?

**Comment by spMohanty** [comment]
@eostermueller : How did you solve this issue ?

**Comment by Aracki** [comment]
Just add `.gif` extension.

**Comment by Ishaan28malik** [comment]
This same error occurred with me, while I was adding an mp4 format file.

**Comment by GaganBhat** [comment]
Same issue with a giphy link.

**Comment by giannif** [comment]
@GaganBhat I'm pretty sure this is new with GIPHY links, they've always worked for me. Now though, I believe there is a restriction on file size:

giphy.gif (7.7MB):
<img src="https://media3.giphy.com/media/xUA7b2iWIZd3e2jyec/giphy.gif"/>

Smaller rendition:
<img src="https://media3.giphy.com/media/xUA7b2iWIZd3e2jyec/200w.gif"/>

**Comment by GaganBhat** [comment]
Yeah you're right, @giannif, that was the issue, looks they updated the camo configuration. 

I fixed my issue by uploading the gif directly into my repo (yes, unfortunately it adds size), and then referring to it with a relative path to the readme.
